features/support/tag_finder.rb

```diff
@@ -0,0 +1,21 @@
+# Finds Git tags, returns them as TagList instances
+class TagFinder
+
+  # Returns all tags (local + remote)
+  def self.all_tags
```

Comment:
Okay, good point about returning TagList instances here. Done.

I'd rather keep type information out of method signatures, and I can't come up with a good name that implies a list without mentioning the type. I have added a description of the return format to the documentation for this method. The `TagFinder` class is also defined as returning TagLists. I think that's enough documentation. What do you think?